R/check.R

```diff
@@ -34,35 +34,37 @@
 check <- function(path = ".", platform = NULL,
                   email = NULL, valgrind = FALSE, check_args = character(),
                   env_vars = character(), show_status = interactive()) {
-
+  
   ## Check that it is a package
   path <- normalizePath(path)
   assert_that(is_pkg_dir_or_tarball(path))
   assert_that(is_flag(valgrind))
   assert_that(is_named(env_vars))
   assert_that(is.character(env_vars))
-
+  
   ## Make sure that maintainer email was validated
   if (is.null(email)) email <- get_maintainer_email(path)
   if (is.na(email)) stop("Cannot get email address from package")
   assert_validated_email_for_check(email)
-
+  
   platform <- match_platform(platform)
-
+  
   ## Build the tar.gz, if needed
   if (file.info(path)$isdir) {
     header_line("Building package")
     pkg_targz <- build_package(path, tmpdir <- tempfile())
   } else {
     pkg_targz <- path
   }
-
+  
   ## Add valgrind to check_args
+  ## Add --no-build-vignettes --no-manual for fedora
   check_args <- c(
     check_args,
+    if (grepl("fedora",platform)) "--no-build-vignettes --no-manual",
```

Comment:
This is not quite good, because `platform` is potentially a vector here. I think we need to change `check_args` so it can be a list if multiple platforms are specified.

Reply:
Is this done anytime soon?

Reply:
Oh, I did not realize you were waiting for me to do something, sorry. :)